interbtc-spec/docs/source/spec/nomination.rst

```diff
@@ -208,463 +140,156 @@ Specification
-
-2. Check if the operator has enough collateral of its own (excluding nominations). If not, throw ``InsufficientCollateral``.
-
-3. Immediately refund, proportionally, nominated collateral that would cause the Max Nomination Ratio to be exceeded.
-
-4. Add the withdrawal request to the ``pendingWithdrawals`` array in the ``Operator`` struct.
-
-5. Decrease the ``backing_collateral`` of ``operatorId`` in the :ref:`Vault-registry`.
-
-
-
-.. _requestNominatorCollateralWithdrawal:
-
-requestNominatorCollateralWithdrawal
-------------------------------------
-
-Request a nominator collateral withdrawal, subject to an unbonding period.
-
-Specification
-.............
-
-*Function Signature*
-
-``requestNominatorCollateralWithdrawal(nominatorId, operatorId, amount)``
-
-*Parameters*
-
-* ``nominatorId``: the id of the requester.
-
-* ``operatorId``: the id of the operator to withdraw from.
-
-* ``amount``: the amount to withdraw.
-
-*Events*
-
-* ``RequestNominatorCollateralWithdrawal(requestId, nominatorId, operatorId, maturity, amount)``
-
-*Errors*
-
-* ``VaultNotOptedInToNomination``: the ``operatorId`` is not an Operator.
-
-* ``NominatorNotFound``: the ``nominatorId`` is not a Nominator.
-
-* ``TooLittleNominatedCollateral``: the caller has requested to withdraw more collateral than it owns.
-
-Preconditions
-.............
-
-* The BTC Parachain status in the :ref:`security` component must be set to ``RUNNING:0``.
-
-
-Function Sequence
-.................
-
-1. Check ``operatorId`` is an operator. If not, throw ``VaultNotOptedInToNomination``.
-
-2. Check ``nominatorId`` is a nominator. If not, throw ``NominatorNotFound``.
-
-3. Check if the caller has at least as much nominated collateral as ``amount``. If not, throw ``TooLittleNominatedCollateral``.
-
-4. Add the withdrawal request to the ``pendingWithdrawals`` array in the ``Nominator`` struct for ``nominatorId``, inside the ``Operator`` struct of ``operatorId``.
-
-5. Decrease the ``backing_collateral`` of ``operatorId`` in the :ref:`Vault-registry`.
-
-
-.. _executeOperatorWithdrawal:
-
-executeOperatorWithdrawal
--------------------------
-
-Execute all matured (unbonded) withdrawal requests of an operator.
-
-Specification
-.............
-
-*Function Signature*
-
-``executeOperatorWithdrawal(operatorId)``
-
-*Parameters*
-
-* ``operatorId``: the id of the requester.
-
-*Events*
-
-* ``ExecuteOperatorCollateralWithdrawal(operatorId, unbondedCollateral)``
-
-*Errors*
-
-* ``VaultNotOptedInToNomination``: the ``operatorId`` is not an Operator.
-
-* ``NoMaturedCollateral``: either no collateral withdrawal has been requested, or the requests have not matured yet.
-
-Preconditions
-.............
+*Postconditions*
 
-* The BTC Parachain status in the :ref:`security` component must be set to ``RUNNING:0``.
+* The Vault's collateral MUST increase by the amount nominated.
+* The nominators balance MUST decrease by the amount nominated.
```

Comment:
I guess also the nominator's staked amount or something must be increased?

Reply:
Yes, but I would reference this when we add the `staking` specification.